app-misc/razergenie/razergenie-9999.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit git-r3 meson
+
+DESCRIPTION="Qt application for configuring your Razer devices under GNU/Linux."
+HOMEPAGE="https://github.com/z3ntu/RazerGenie"
+EGIT_REPO_URI="https://github.com/z3ntu/RazerGenie.git"
+EGIT_CLONE_TYPE="shallow"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux"
+
+RDEPEND="
+	app-misc/openrazer[daemon]
+	app-misc/razer-test
```

Comment:
Why does this depend on razer_test (supposed to be an openrazer replacement) and on openrazer at the same time?

Reply:
Both are available as an option in the gui. Unfortunately the razer_test backend isn't easy to decouple from current Razer Genie build.

Reply:
Hmm.. I think we should make openrazer optional via a USE flag, then. Either I can do it after merging or you can.

Reply:
Making either of those optional will require additional src_prepare trickery as there are no direct build system switches to disable them. I could prepare them, but idk how much time it'll take me and when.

Feel free to take any approach you like.

Reply:
Argh! Yeah, seems like those are actually hard deps.
If they are lacking, it'll fall back to a git clone, I think. So not what we want.

I guess we'll just leave it in for now, see how its behaviour changes in the upstream project.